detector/detection_results.go

```diff
@@ -295,24 +296,74 @@ func (r *DetectionResults) Report() string {
 		fmt.Printf("\n\x1b[1m\x1b[31mTalisman Report:\x1b[0m\x1b[0m\n")
 		table.AppendBulk(data)
 		table.Render()
-		result = result + fmt.Sprintf("\n\x1b[33mIf you are absolutely sure that you want to ignore the above files from talisman detectors, consider pasting the following format in .talismanrc file in the project root\x1b[0m\n")
-		result = result + r.suggestTalismanRC(filePathsForIgnoresAndFailures)
-		result = result + fmt.Sprintf("\n\n")
+		r.suggestTalismanRC(fs, ignoreFile, filePathsForIgnoresAndFailures, promptContext)
 	}
 	return result
 }
 
-func (r *DetectionResults) suggestTalismanRC(filePaths []string) string {
-	var fileIgnoreConfigs []FileIgnoreConfig
+func (r *DetectionResults) suggestTalismanRC(fs afero.Fs, ignoreFile string, filePaths []string, promptContext prompt.PromptContext) {
+	var entriesToAdd []FileIgnoreConfig
+
 	for _, filePath := range filePaths {
 		currentChecksum := utility.CollectiveSHA256Hash([]string{filePath})
 		fileIgnoreConfig := FileIgnoreConfig{filePath, currentChecksum, []string{}}
-		fileIgnoreConfigs = append(fileIgnoreConfigs, fileIgnoreConfig)
+		if promptContext.Interactive{
+			if confirm(fileIgnoreConfig, promptContext) {
+				entriesToAdd = append(entriesToAdd, fileIgnoreConfig)
+			}
+			addToTalismanIgnoreFile(entriesToAdd, fs, ignoreFile)
+		} else {
+			entriesToAdd = append(entriesToAdd, fileIgnoreConfig)
```

Comment:
Thats how it is now.

Reply:
I deleted my .talismanrc and did `./talisman_darwin_amd64 -g pre-commit -i`.
Content of .talismanrc
```yaml
fileignoreconfig:
- filename: vendor/github.com/davecgh/go-spew/spew/dump.go
  checksum: 5b8a26e774b93cc00b7a1e4c532fbf84583ca678e1ec9d5f23b8e6391a845372
  ignore_detectors: []
scopeconfig: []
fileignoreconfig:
- filename: vendor/github.com/davecgh/go-spew/spew/dump.go
  checksum: 5b8a26e774b93cc00b7a1e4c532fbf84583ca678e1ec9d5f23b8e6391a845372
  ignore_detectors: []
- filename: vendor/golang.org/x/sys/unix/env_unix.go
  checksum: 9d2996cad59e959a271df39789348a714a062f251424ab558a302a08c68fb11f
  ignore_detectors: []
scopeconfig: []
fileignoreconfig:
- filename: vendor/github.com/davecgh/go-spew/spew/dump.go
  checksum: 5b8a26e774b93cc00b7a1e4c532fbf84583ca678e1ec9d5f23b8e6391a845372
  ignore_detectors: []
- filename: vendor/golang.org/x/sys/unix/env_unix.go
  checksum: 9d2996cad59e959a271df39789348a714a062f251424ab558a302a08c68fb11f
  ignore_detectors: []
- filename: vendor/golang.org/x/sys/unix/affinity_linux.go
  checksum: e41d86cd25999d19bdec3453dad8ca86b9a16ede4e190649dd953885f7f13131
  ignore_detectors: []
scopeconfig: []
fileignoreconfig:
- filename: vendor/github.com/davecgh/go-spew/spew/dump.go
  checksum: 5b8a26e774b93cc00b7a1e4c532fbf84583ca678e1ec9d5f23b8e6391a845372
  ignore_detectors: []
- filename: vendor/golang.org/x/sys/unix/env_unix.go
  checksum: 9d2996cad59e959a271df39789348a714a062f251424ab558a302a08c68fb11f
  ignore_detectors: []
- filename: vendor/golang.org/x/sys/unix/affinity_linux.go
  checksum: e41d86cd25999d19bdec3453dad8ca86b9a16ede4e190649dd953885f7f13131
  ignore_detectors: []
scopeconfig: []
```

There is duplication of content. Could be because of the above code. Could you check?

Reply:
@leoOrion Can you check this